.env.template

```diff
@@ -29,9 +29,7 @@ POSTGRESQL_PASSWORD=content
 POSTGRESQL_PORT=5432
 RESQUE_NAMESPACE=resque:development
 SOUNDCLOUD_CLIENT_ID=
-SWAP_DOCS=
-SWAP_DOCS_LATEX=
-UNBOUNDED_DOMAIN=unbounded.org
+APPLICATION_DOMAIN=unbounded.org
```

Comment:
maybe this should be example.org instead, just to be neutral?

Reply:
Agree. Finishing 2nd commit containing more cleanup. Placed `localhost` there for easy local development.